docs/source/markdown/podman-kube-down.1.md

```diff
@@ -0,0 +1,30 @@
+% podman-kube-down(1)
+
+## NAME
+podman-kube-down - Remove containers or pods based on Kubernetes YAML
+
+## SYNOPSIS
+**podman kube down** *file.yml|-*
+
+## DESCRIPTION
+**podman kube down** will read in a structured file of Kubernetes YAML. It will then tear down any pods that were based on the specifications described in the YAML and created using `podman kube play`. Any volumes that were created will remain intact. If the YAML file is specified as "-" then `podman kube down` will
+read the YAML from stdin.
+
+Note: The command `podman kube play --down` will perform the same operation as this command.
```

Comment:
Please drop this sentence.  If we add new flags to `kube down` in the future, the sentence would not hold anymore.